Source/Atomic/Scene/Node.h

```diff
@@ -495,7 +495,7 @@ class ATOMIC_API Node : public Animatable
     void RemoveChild(Vector<SharedPtr<Node> >::Iterator i);
     /// Return child nodes recursively.
     void GetChildrenRecursive(PODVector<Node*>& dest) const;
-    /// Return child nodes with a specific component recursively.
+	/// Return child nodes with a specific component recursively.
     void GetChildrenWithComponentRecursive(PODVector<Node*>& dest, StringHash type) const;

```

Comment:
Ok, yeah I changed those directly on GH (oddly the tab shows as spaces in
VS, and deleting the whitespace and re-adding it didn't show in git either)

Regards,
Matt Benic
mattbenic@gmail.com | +27 (72) 1826161 | za.linkedin.com/in/mattbenic

On Thu, Jun 11, 2015 at 5:12 PM, JoshEngebretson <notifications@github.com>
wrote:

> In Source/Atomic/Scene/Node.h
> <https://github.com/AtomicGameEngine/AtomicGameEngine/pull/109#discussion_r32231077>
> :
>
> > @@ -495,7 +495,7 @@ class ATOMIC_API Node : public Animatable
> >      void RemoveChild(Vector<SharedPtr<Node> >::Iterator i);
> >      /// Return child nodes recursively.
> >      void GetChildrenRecursive(PODVector<Node*>& dest) const;
> > -    /// Return child nodes with a specific component recursively.
> > +	/// Return child nodes with a specific component recursively.
> >      void GetChildrenWithComponentRecursive(PODVector<Node*>& dest, StringHash type) const;
>
> Yeah, as actually doing a code review here should probably flag it :) It
> is pretty easy to spot these when submitting a PR.
>
> You don't have to recreate the PR, if you commit to the PR branch the
> changes will show up in the PR
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AtomicGameEngine/AtomicGameEngine/pull/109/files#r32231077>
> .
>